VersionControlUI/src/items/VDiffComparisonPair.cpp

```diff
@@ -50,6 +52,127 @@ void VDiffComparisonPair::initializeForms()
 	auto newVersion = item(&I::newVersionNode_, [](I* v) {
 			return v->node()->newVersionNode();});
 
+	auto componentType = item<Visualization::VText>(&I::componentType_, [](I* v) {
+			return v->node()->componentType();},
+			[](I* v) {return &v->style()->componentType();});
+
+	auto noNodeFound = item<Visualization::Static>(&I::nodeNotFoundIcon_, &StyleType::nodeNotFoundIcon);
+
+
+	auto objectPath = item<Visualization::VText>(&I::singleObjectPath_, [](I* v) {
+			return v->node()->singleObjectPath();},
+			[](I* v) {return &v->style()->singleObjectPath();});
+
+	// form with both nodes and only one object path
+
+	auto infoGrid = (new Visualization::GridLayoutFormElement{})
+			->setHorizontalSpacing(50)
+			->setLeftMargin(10)
+			->setRightMargin(10)
+			->setHorizontalAlignment(Visualization::LayoutStyle::Alignment::Center)
+			->setNoBoundaryCursors([](Item*){return true;})->setNoInnerCursors([](Item*){return true;})
+			->setColumnStretchFactor(0, 1)
+			->put(0, 0, objectPath)
+			->put(0, 1, componentType);
+
+	auto diffGrid = (new Visualization::GridLayoutFormElement{})
+			->setHorizontalSpacing(50)
+			->setLeftMargin(10)
+			->setRightMargin(10)
+			->setHorizontalAlignment(Visualization::LayoutStyle::Alignment::Center)
+			->setNoBoundaryCursors([](Item*){return true;})->setNoInnerCursors([](Item*){return true;})
+			->setColumnStretchFactor(0, 1)
+			->setColumnStretchFactor(1, 1)
+			->setRowStretchFactor(0, 1)
+			->setColumnHorizontalAlignment(0, Visualization::LayoutStyle::Alignment::Right)
+			->setColumnHorizontalAlignment(1, Visualization::LayoutStyle::Alignment::Left)
+			->put(0, 0, oldVersion)
+			->put(1, 0, newVersion);
+
+	auto container = (new Visualization::GridLayoutFormElement{})
+				->setHorizontalSpacing(30)
```

Comment:
These lines are indented with one additional tab compared to the analogous lines in the form elements above. Please use consistent indentation.